pallets/creditcoin/src/migrations/v1.rs

```diff
@@ -181,90 +182,113 @@ type DealOrders<T: crate::Config> = StorageDoubleMap<
 	DealOrder<AccountIdOf<T>, BlockNumberOf<T>, HashOf<T>, MomentOf<T>>,
 >;
 
-pub(crate) fn migrate<T: Config>() -> Weight {
-	let mut weight: Weight = Weight::zero();
+pub(crate) struct Migration<Runtime>(pub PhantomData<Runtime>);
 
-	let weight_each = T::DbWeight::get().reads_writes(1, 1);
-	AskOrders::<T>::translate::<
-		OldAskOrder<AccountIdOf<T>, BlockNumberOf<T>, HashOf<T>, MomentOf<T>>,
-		_,
-	>(|_expiration, _hash, ask| {
-		weight = weight.saturating_add(weight_each);
-		Some(AskOrder {
-			block: ask.block,
-			blockchain: ask.blockchain,
-			expiration_block: ask.expiration_block,
-			lender: ask.lender,
-			lender_address_id: ask.lender_address_id,
-			terms: AskTerms(LoanTerms {
-				amount: ask.terms.0.amount,
-				interest_rate: InterestRate {
-					rate_per_period: ask.terms.0.interest_rate,
-					decimals: OLD_INTEREST_RATE_DECIMALS,
-					period: Duration::from_millis(ask.terms.0.maturity.unique_saturated_into()),
-				},
-				term_length: Duration::from_millis(ask.terms.0.maturity.unique_saturated_into()),
-			}),
-		})
-	});
+impl<Runtime: Config> Migration<Runtime> {
+	pub(super) fn new() -> Self {
+		Self(PhantomData::<Runtime>)
```

Comment:
```suggestion
		Self(PhantomData)
```
As much as I love a good [turbofish](https://doc.rust-lang.org/book/appendix-02-operators.html?highlight=turbofish#non-operator-symbols), it's not necessary here (and it's kinda noisy syntax)